api/python/cellxgene_census/tests/experimental/pp/test_stats.py

```diff
@@ -0,0 +1,144 @@
+import math
```

Comment:
I noticed that test_online.py still exists, which tests MeanVarianceAccumulator directly.  

Suggest adding a test for MeanAccumulator in that file so you directly test the lower level implementation.